upload.py

```diff
@@ -1404,7 +1404,9 @@ def GetStatus(self, filename):
       # If file is in a cl, the output will begin with
       # "\n--- Changelist 'cl_name':\n".  See
       # http://svn.collab.net/repos/svn/trunk/notes/changelist-design.txt
-      if (len(status_lines) == 3 and
+      # The file could also be moved and in a CL, in which case there will
+      # be 4 lines in the output, the last one starting with "> moved to"
+      if (len(status_lines) >= 3 and
```

Comment:
Are there other cases where it could be more than 4 lines? Otherwise I'd prefer it more explicit here, like `len(status_lines) in (3, 4)`

Reply:
I don't know.  I encountered only this particular case so I don't know if there's any other cases that happen in the wild.  I changed it to (3, 4)